Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-add/user-add.component.ts

```diff
@@ -71,31 +72,34 @@ export class UserAddComponent {
     }
 
     checkEmail(email) {
-        this.isEmailExist = false;
-
-        this.userService.findUserByEmail(email + "@promactinfo.com").subscribe((isEmailExist) => {
-            if (isEmailExist) {
-                this.isEmailExist = true;
-            }
-            else {
-                this.isEmailExist = false;
+        if (email !== "" && email !== undefined) {
+            this.isEmailExist = false;
+            this.userService.findUserByEmail(email + "@promactinfo.com").subscribe((isEmailExist) => {
+                if (isEmailExist) {
```

Comment:
This entire block can be written in single line. Additionally naming is confusing here. Why are you using the same name for a local variable and instance variable? I think this block is redundant.